### PR TITLE
Fix NullPointerException at JmxScraper.java:240

### PR DESCRIPTION
**Pull request by juliohm1978**
Follow up to #252. This PR fixes a NullPointerException at JmxScraper.java:240.

### CODE REVIEW

File: collector/src/main/java/io/prometheus/jmx/JmxScraper.java [hunk 1]
```diff
@@ -237,7 +237,9 @@ private void processBeanValue(
                 if (valu instanceof CompositeData) {
                     CompositeData composite = (CompositeData) valu;
                     for (String idx : rowKeys) {
-                        l2s.put(idx, composite.get(idx).toString());
+                        if ( composite.get(idx) != null ) {
```
Comment (brian-brazil) [inline collector/src/main/java/io/prometheus/jmx/JmxScraper.java]: You don't need the whitespace inside the parens

File: collector/src/main/java/io/prometheus/jmx/JmxScraper.java [hunk 2]
```diff
@@ -237,7 +237,9 @@ private void processBeanValue(
...
+                            l2s.put(idx, composite.get(idx).toString());
```
Comment (brian-brazil) [inline collector/src/main/java/io/prometheus/jmx/JmxScraper.java]: You've a race here, this get might return null.

File: collector/src/main/java/io/prometheus/jmx/JmxScraper.java [hunk 3]
```diff
@@ -237,7 +237,9 @@ private void processBeanValue(
                 if (valu instanceof CompositeData) {
                     CompositeData composite = (CompositeData) valu;
                     for (String idx : rowKeys) {
-                        l2s.put(idx, composite.get(idx).toString());
+                        if ((composite != null) && (composite.get(idx) != null)) {
```
Comment (brian-brazil) [inline collector/src/main/java/io/prometheus/jmx/JmxScraper.java]: I don't see how composite can be null here, it'd have failed the instanceof. What you want to do is call `composite.get(idx)` only once.

Comment (juliohm1978) [reply collector/src/main/java/io/prometheus/jmx/JmxScraper.java]: You are right. I misunderstood your last comment. Fixing it now.

**Comment by brian-brazil** [comment]
Thanks!

**Pull request merged**